tests/checkers/inference/test/CFInferenceTest.java

```diff
@@ -39,6 +39,10 @@ public String getPathToAfuScripts() {
         return System.getProperty("path.afu.scripts");
     }
 
+    public String getPathToInferenceScript() {
+        return "./scripts/inference";
```

Comment:
Yes, moving this part into configuration script still be better. Will change this.